src/components/NewTable/components/TablePagination/components/TablePaginationButton.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { css, cx } from '@emotion/css';
+import { tokens } from '../../../../../theme/tokens';
+
+interface TablePaginationArrowProps extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'onClick'> {
+  active?: boolean;
+  onClick?: (page?: number) => void;
+}
+export const TablePaginationButton = ({ active, children, onClick, ...props }: TablePaginationArrowProps) => {
+  return (
+    <button
+      onClick={!!onClick ? () => onClick() : undefined}
```

Comment:
```suggestion
      onClick={() => onClick && onClick()}
```
Wdyt. Would work?

Reply:
It's not ideal.
In this case: `onClick={() => onClick && onClick()}`, even if our `onClick` from the props is `undefined` the button will still have attached the `onClick` listener, which will get fired when we click it (and do nothing).

In the original case, if there's no `onClick` coming from the props the button will not even listen for clicks on it.